upload-api/functions/ucan-invocation-router.js

```diff
@@ -118,6 +118,17 @@ export const knownWebDIDs = {
  * @param {import('aws-lambda').APIGatewayProxyEventV2} request
  */
 export async function ucanInvocationRouter(request) {
+  // Capture X-Client custom header for analytics - ignore if not present
+  const clientId = request.headers['X-Client']
+  if (clientId) {
+    console.log(JSON.stringify({
+      message: 'Client request',
+      clientId,
+      requestId: request.requestContext?.requestId || 'unknown',
+      timestamp: new Date().toISOString()
+    }))
+  }
```

Comment:
I would just always log a value so that we can easily count known values as a percentage of all requests.

```suggestion
  // Capture X-Client custom header for analytics
  const clientId = request.headers['X-Client'] ?? 'Storacha/?'
  console.log(JSON.stringify({
    message: 'Client request',
    clientId,
    requestId: request.requestContext?.requestId || 'unknown',
    timestamp: new Date().toISOString()
  }))
```